frontend/src/theme/selectStyle.tsx

```diff
@@ -0,0 +1,55 @@
+import colors from "./colors";
+
+const optionBackgroundColor = (isSelected: boolean, isFocused: boolean) => {
+  if (isSelected) {
+    return colors.teal[100];
+  }
+  if (isFocused) {
+    return colors.teal[50];
+  }
+  return colors.white.default;
+};
+
+const selectStyle: any = {
+  singleValue: (provided: any) => ({
+    ...provided,
+    color: colors.gray[500],
+    borderColor: colors.gray[200],
+    boxShadow: "none",
+  }),
+  control: (provided: any, state: any) => ({
+    ...provided,
+    backgroundColor: state.isDisabled ? colors.teal[50] : colors.white.default,
+    borderColor: colors.gray[200],
+    boxShadow: "none",
+    borderRadius: "4px",
+    "&:hover": {
+      border: `1px solid ${colors.gray[200]}`,
+      cursor: "pointer",
+    },
```

Comment:
nit: The same way we extracted out the `optionBackgroundColor` function, could we extract out the `multiValue`, `singleValue`, `control`, and `option` methods? Will make this file a bit more consistent

Reply:
```suggestion
  const controlStyle = (provided: any, state: any) => ({
    ...provided,
    backgroundColor: state.isDisabled ? colors.teal[50] : colors.white.default,
    borderColor: colors.gray[200],
    boxShadow: "none",
    borderRadius: "4px",
    "&:hover": {
      border: `1px solid ${colors.gray[200]}`,
      cursor: "pointer",
    },
    })

  control: controlStyle(provided, state),
```